.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: java
-jdk: oraclejdk7
+jdk: oraclejdk8
```

Comment:
Yea, I changed the version that Travis CI uses to avoid the CI job failure.
https://travis-ci.org/muga/embulk-filter-calcite/builds/228161762#L438
```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':checkstyleMain'.
> com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask : Unsupported major.minor version 52.0
```

But, of course that's not good. Let me try to find workaround.